examples/mld/mld_main.c

```diff
@@ -258,9 +258,10 @@ int main(int argc, FAR char *argv[])
   FAR char *iobuffer = NULL;
   struct sockaddr_in6 host;
 #ifdef CONFIG_NET_ROUTE
-  struct sockaddr_in6 target;
-  struct sockaddr_in6 router;
-  struct sockaddr_in6 netmask;
+  struct sockaddr_storage target;
```

Comment:
Changing to `struct sockaddr_storage` in this block leads to quite a significant stack increase. Do we need to change application default stack value or the current value is sufficient?

Reply:
The default stack is 2KB which should be enough. BTW, since printf consume the stack around 256bytes, so the increase from sockaddr_storage(~384 bytes) is safe.